yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1319,7 +1319,31 @@ func (yb *TargetYugabyteDB) ClearMigrationState(migrationUUID uuid.UUID, exportD
 	return nil
 }
 
+func (yb *TargetYugabyteDB) GetMissingImportSchemaPermissions() ([]string, error) {
+	// check if the user is a superuser
+	isSuperUser, err := yb.checkIfCurrentUserIsSuperUser()
+	if err != nil {
+		return nil, fmt.Errorf("checking if user is superuser: %w", err)
+	}
+	if !isSuperUser {
+		errorMsg := fmt.Sprintf("User %s is not a superuser.", yb.tconf.User)
+		return []string{errorMsg}, nil
+	}
+
+	return nil, nil
+}
+
 func (yb *TargetYugabyteDB) GetMissingImportDataPermissions() ([]string, error) {
+	// check if the user is a superuser
+	isSuperUser, err := yb.checkIfCurrentUserIsSuperUser()
+	if err != nil {
+		return nil, fmt.Errorf("checking if user is superuser: %w", err)
+	}
+	if !isSuperUser {
+		errorMsg := fmt.Sprintf("User %s is not a superuser.", yb.tconf.User)
+		return []string{errorMsg}, nil
+	}
+
```

Comment:
Yeah I have removed the function GetMissingImportSchemaPermissions as discussed by Aneesh above since we anyways require it only for YB. I have added it to importSchema.go only. Can you have a look there?